CHANGELOG.md

```diff
@@ -44,47 +48,14 @@ The types of changes are:
 
 * Allow multiple data uses as long as their processing activity name is different [#2905](https://github.com/ethyca/fides/pull/2905)
 * use HTML property, not text, when dispatching Mailchimp Transactional emails [#2901](https://github.com/ethyca/fides/pull/2901)
-* Allow multiple data uses as long as their processing activity name is different [#2905](https://github.com/ethyca/fides/pull/2905)
-* Remove policy key from Privacy Center submission modal [#2912](https://github.com/ethyca/fides/pull/2912)
-
-## [2.9.1](https://github.com/ethyca/fides/compare/2.9.0...2.9.1)
```

Comment:
This looks like I've taken a lot out — `2.9.1` was duplicated in the original changelog.